web/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "gallery",
+    "short_name": "gallery",
+    "start_url": ".",
+    "display": "minimal-ui",
+    "background_color": "#0175C2",
+    "theme_color": "#0175C2",
+    "description": "A new Flutter project.",
+    "orientation": "portrait-primary",
+    "prefer_related_applications": false,
+    "icons": [
+        {
+            "src": "icons/Icon-192.png",
```

Comment:
Not necessary for this PR, but we should update all of these new assets to be the proper ones

Reply:
OK, what are the proper ones?

Reply:
Let me get you some images